README.md

```diff
@@ -71,7 +71,7 @@ Much of the philosophy behind Bedrock is inspired by the [Twelve-Factor App](htt
    ```sh
    $ composer create-project roots/bedrock
    ```
-2. Update environment variables in the `.env` file. Wrap values that may contain non-alphanumeric characters with quotes, or they may be incorrectly parsed.
+2. Create a `.env` config file (using `.env.example` as a guide) and set your config values. Wrap values that may contain non-alphanumeric characters with quotes, or they may be incorrectly parsed.
```

Comment:
Might be we can follow suite with the first item in this list, and give the code for this as well, for quick copy/pasting.

E.G.:

2. Create a `.env` config file (using `.env.example` as a guide) and set your config values. Wrap values that may contain non-alphanumeric characters with quotes, or they may be incorrectly parsed.
   ```sh
   $ cp .env.example .env
   ```

Reply:
Yeah I had considered that, I'll add it 👍